playgroundFaq.md

```diff
@@ -0,0 +1,66 @@
+2. **VC-API exchanges workflow:** in this workflow, the playground site uses CHAPI to communicate the URL of a VC-API/exchanges endpoint where a Verifiable Credential can be picked up.  The use of CHAPI here preserves the individual's choice of wallet and avoids the need to make assumptions about credential types, wallet apps, etc.  The wallet then hits the specified /exchanges endpoint and receives a Verifiable Presentation containing the Credential.  DID Authentication and credential pickup are performed in the same step.  This is especially useful for native mobile apps.
+
+### Integrating with the basic CHAPI store() workflow
+Any CHAPI-enabled wallet can work with the CHAPI-playground.  There is no explicit setting or integration required with the CHAPI-playground site; all that’s required is that the user register the wallet with their browser (web wallets) or as a Share Target (native mobile apps).  See the [developer notes](https://chapi.io/developers/wallets) for instructions and examples on how to implement CHAPI in a web wallet.
+
+When the CHAPI-playground issues a credential, it generates a CHAPI `store` event.  This is also documented at the link above.
+
+### Performing DID Authentication (“DID Auth”) with the CHAPI-playground in the store() workflow?
+
+[Start here](https://gist.github.com/evanlally/3feb599ceadecc511a8a35d1a3b306b1)
+
+These are examples of the types of data passed between the CHAPI playground and the Veres Wallet during the DID Auth workflow.  The CHAPI-playground will send the Verifiable Presentation Request in (1.) as a "web" credential object using `navigator.credentials.get()`.
+
+Your wallet will respond with the Verifiable Presentation in (2.), also as a "web" credential object.  The playground receives the DID Auth, it extracts the value of `holder` and inserts it into the credential being issued.  Then, the credential is issued using the basic CHAPI `store` workflow.
+
+All of this follows the patterns in the [VPR spec](https://w3c-ccg.github.io/vp-request-spec/#example-example-get-request).
+
+### Integrating with the VC-API exchanges workflow
+Here's an overview of the steps:
+1. CHAPI Playground sends a DID Auth Request (VPR) over CHAPI to the wallet.  This includes an interact field that specifies an \exchanges endpoint on the CHAPI Playground.
+2. The Wallet POSTs a VP containing the DID Auth to the \exchanges endpoint.
+3. The CHAPI Playground \exchanges endpoint responds with a VP containing the specified credential
+
+Example data for each step can be found [here](https://gist.github.com/tolson4/6e44720a15957136fbe6ca33403c4bcb).
+
+We’ve added an example to the [Verifiable Presentation Request (VPR) spec](https://w3c-ccg.github.io/vp-request-spec/#example-example-interact-request) to show how an issuer can ask for DID Authentication and then point the wallet at /exchanges endpoint to perform the Auth-Pickup workflow.
```

Comment:
Consider how to remove the "we" tense in this document.  IMO, "you" is OK, but "we" is a little confusing.